internal/utils.go

```diff
@@ -0,0 +1,36 @@
+package internal
+
+import (
+	"math/rand"
+	"strings"
+)
+
+func RepeatSlice(slice []string, n int) []string {
```

Comment:
Haven't checked usages yet, but feels weird to have one method public and others private